### DISCUSSION TITLE
vscode-remote-ssh

### DISCUSSION THREAD

**Discussion by giscus** [AMD-NICK/blog.amd-nick.me#28] [Blog comments]
# vscode-remote-ssh

official banner

https://blog.amd-nick.me/vscode-remote-ssh

**Comment by S1ROZHA** [comment]
Прикольная штука, но изредка с ней бывают такие лютые траблы когда загружает проц и хард на фулл и пизда серваку пока не вырубишь.
Потыкай pm2 deploy тебе понравиться)

**Reply by shockpast** [reply]
Ниразу не заметил такого за 2 года юзания 🤔

**Reply by AMD-NICK** [reply]
я пока не сталкивался с такой проблемой. Думаю, если бы она была широко распространена, то об этом везде бы писалось и это фиксили бы

**Reply by AMD-NICK** [reply]
> изредка с ней бывают такие лютые траблы когда загружает проц и хард на фулл и пизда серваку пока не вырубишь

Случайно снова попал на этот пост и заметил твой комментарий. Ты знаешь, а ведь действительно на одном из хостов у меня аномально время от времени идет 100% ЦП, когда я подключаюсь по Remote SSH плагину. И приходится либо долго ждать, пока пройдет, либо ребутать сервер.

Как-то решал проблему?

**Comment by sambatya1** [comment]
Чел, спасибо тебе за статью.

**Reply by AMD-NICK** [reply]
❤️
